silkworm/rpc/core/logs_walker.cpp

```diff
@@ -149,45 +149,50 @@ Task<void> LogsWalker::get_logs(BlockNum start,
     Logs filtered_chunk_logs;
 
     uint64_t block_timestamp{0};
+    std::shared_ptr<BlockWithHash> block_with_hash{};
```

Comment:
No need for `{}` to initialise empty `std::shared_ptr`

Reply:
ok